pkgs/tools/networking/unbound/default.nix

```
@@ -58,6 +58,29 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-vB1Xbz3YRqBzmtxB/6pwJATGdn0rYILeufL5fLsko6k=";
   };
 
+
+  # Cherry pick some already merged upstream patches for configure
+  patches = [
+    # Search for protobuf-c with pkg-config
+    # https://github.com/NLnetLabs/unbound/pull/999
+    (fetchpatch {
+      url = "https://github.com/NLnetLabs/unbound/commit/59d98b9ef64e597c331c27160715d7a1b40c8638.patch";
+      hash = "sha256-DvYoYBTXOwbR8Z0GRgt724WqX3dbIEOdICU2/VMRSVQ=";
+    })
+    # Fix for previous patch
+    # https://github.com/NLnetLabs/unbound/issues/1006
+    (fetchpatch {
+      url = "https://github.com/NLnetLabs/unbound/commit/93490a0fc1bf9e62e6edcd6b69f1463c7ac410e9.patch";
```

```suggestion
      url = "https://github.com/NLnetLabs/unbound/commit/93490a0fc1bf9e62e6edcd6b69f1463c7ac410e9.patch";
      excludes = [ "doc/Changelog" ];
```

```
building '/nix/store/bhk94rq5bxgva0h97p8yaaqvp5va1k2m-unbound-1.19.1.drv'...
Running phase: unpackPhase
unpacking source archive /nix/store/dah45yk4pmsxgvs0fbzhaq1g25h08m4y-unbound-1.19.1.tar.gz
source root is unbound-1.19.1
setting SOURCE_DATE_EPOCH to timestamp 1707825929 of file unbound-1.19.1/util/configparser.c
Running phase: patchPhase
applying patch /nix/store/8maj3xnkf65v47iqcm3ijy7acnpxlf1y-59d98b9ef64e597c331c27160715d7a1b40c8638.patch
patching file dnstap/dnstap.m4
applying patch /nix/store/7irj3hhkccx3x7r7zdk4aggkzfg0kflm-93490a0fc1bf9e62e6edcd6b69f1463c7ac410e9.patch
patching file configure.ac
patching file dnstap/dnstap.m4
patching file doc/Changelog
Hunk #1 FAILED at 1.
1 out of 1 hunk FAILED -- saving rejects to file doc/Changelog.rej
error: builder for '/nix/store/bhk94rq5bxgva0h97p8yaaqvp5va1k2m-unbound-1.19.1.drv' failed with exit code 1;
```

also please update the hash

---

sorry... done and checked compilation on master branch